templates/shortcode-receipt.php

```diff
@@ -146,14 +146,14 @@
 						$download_files = edd_get_download_files( $item['id'], $price_id );
 						?>
 
-						<div class="edd_purchase_receipt_product_name">
+						<div class="edd_purchase_receipt_product_name">§
 							<?php echo esc_html( $item['name'] ); ?>
 							<?php if ( edd_has_variable_prices( $item['id'] ) && ! is_null( $price_id ) ) : ?>
 							<span class="edd_purchase_receipt_price_name">&nbsp;&ndash;&nbsp;<?php echo esc_html( edd_get_price_option_name( $item['id'], $price_id, $payment->ID ) ); ?></span>
 							<?php endif; ?>
 						</div>
 
-						<?php if ( $edd_receipt_args['notes'] ) : ?>
+						<?php if ( $edd_receipt_args['notes'] && ! empty( edd_get_product_notes( $item['id'] ) ) ) : ?>
```

Comment:
`edd_get_product_notes()` must be moved outside of `empty()` or `empty()` removed entirely to prevent a fatal error on some PHP versions.